.github/CHANGELOG.md

```diff
@@ -7,29 +7,37 @@
 
 ### Breaking changes
 
-- Deprecate Python 3.7 wheels.
+* Deprecate Python 3.7 wheels.
 [(#75)](https://github.com/PennyLaneAI/pennylane-lightning-gpu/pull/75)
 
 ### Improvements
 
-- Add support for CI checks.
+* `lightning.gpu` is decoupled from Numpy layer during initialization and execution
+and change lightning.gpu to inherit from QubitDevice instead of LightningQubit.
+[(#70)](https://github.com/PennyLaneAI/pennylane-lightning-gpu/pull/70)
+
+* Update `LightningGPU` device following changes in `LightningQubit` inheritance from `DefaultQubit` to `QubitDevice`.
+[(#70)](https://github.com/PennyLaneAI/pennylane-lightning/pull/70)
```

Comment:
Hey @multiphaseCFD! This one should have stayed under Release 0.27, and should be:
```suggestion
* Update `LightningGPU` device following changes in `LightningQubit` inheritance from `DefaultQubit` to `QubitDevice`.
[(#74)](https://github.com/PennyLaneAI/pennylane-lightning-gpu/pull/74)
```
Am I right?  :slightly_smiling_face:

Reply:
correct!